openedx_external_enrollments/tests/tests_factory.py

```diff
@@ -0,0 +1,29 @@
+"""Tests factory file."""
```

Comment:
Is this the same test? Did you change something? @andrey-canon

Reply:
The code is the same, the only thing that has changed is the import paths

Reply:
Ok.